blockchain/test/TabletopDiceNFT.spec.ts

```diff
@@ -103,7 +103,7 @@ describe("TabletopDiceNFT", () => {
       let tokenId = 1;
       let numRolls = 10;
       for (var index = 0; index < numRolls; ++index) {
-        let rollResult = await deployedContract.roll(tokenId);
+        await ethers.provider.send('evm_mine', []); // Advance block number
```

Comment:
I was wondering what this was when you were describing it... It looks like this triggers the test runner to advance the fake blockchain one block. Groovy.